packages/components/src/tooltip/test/index.js

```diff
@@ -23,171 +28,136 @@ describe( 'Tooltip', () => {
 	describe( '#render()', () => {
 		it( 'should not render the tooltip if multiple children are passed', () => {
 			render(
-				<Tooltip text="Help text">
-					<button>Button 1</button>
-					<button>Button 2</button>
+				<Tooltip { ...props }>
+					<Button>Button 1</Button>
+					<Button>Button 2</Button>
 				</Tooltip>
 			);
 
-			const button = screen.getByText( 'Button 1' );
-			act( () => button.focus() );
```

Comment:
Did we intentionally remove the focus here without adding an `await user.tab()` or any other suitable replacement?

Reply:
tl;dr, it was somewhat intentionally unintentional. 😅 But in this case, it does make sense to ensure the button has focus through some `userEvent` method, so I will make that change. 

I had removed it [after reading this](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role#accessibility_concerns):
>If the information is important enough for a tooltip, isn't it important enough to always be visible? 

So I was just thinking about if the tooltip should render (when there is just one child) even when the anchor isn't in focus but stay hidden until its conditions are met. I think in most cases, it's just a visual duplicate of an aria-label, so it shouldn't be needed in the DOM. But then our Storybook example shows a default use case where the tooltip has more information. And in this test, the original tooltip text was called 'help text' which is why I changed it to 'tooltip text' because I think it could be misleading to refer to it as help text or additional info etc.

So the way it renders likely shouldn't change, but maybe the examples and documentation to make sure the use case for a tooltip is clear.